model/src/main/scala/org/broadinstitute/dsde/rawls/StringUtils.scala

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.dsde.rawls
+
+import org.broadinstitute.dsde.rawls.model.{Attributable, AttributeName, ErrorReport, ErrorReportSource}
+import spray.http.StatusCodes
+
+trait StringUtils {
```

Comment:
Instead of making this a trait with an `implicit val`, what about making `StringUtils` an `object` and pass the implicit `ErrorReportSource` to each method? E.g.:
```
def validateUserDefinedString(s: String)(implicit errorReportSource: ErrorReportSource): Unit = // etc
```
IMO that's slightly more idiomatic Scala (maybe I'm biased against OO).

Also `StringUtils` sounds a little general, maybe name this `StringValidationUtils`?